test/common/upstream/load_balancer_fuzz.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "envoy/config/cluster/v3/cluster.proto";
+import "google/protobuf/empty.proto";
+
+message UpdateHealthFlags {
+  // The host priority determines what host set within the priority set which will get updated.
+  uint64 host_priority = 1;
+  // These will determine how many hosts will get placed into health hosts, degraded hosts, and
+  // excluded hosts from the full host list.
+  uint32 num_healthy_hosts = 2;
+  uint32 num_degraded_hosts = 3;
+  uint32 num_excluded_hosts = 4;
+  // This is used to determine which hosts get marked as healthy, degraded, and excluded.
+  bytes random_bytestring = 5 [(validate.rules).bytes = {min_len: 1, max_len: 256}];
```

Comment:
FWIW, an idea of making this run-length encoded just occurred to me, it would be space efficient, but not sure how that would interact with the fuzzing cross-over.